tests/php/Centreon/Application/Controller/PlatformTopologyControllerTest.php

```diff
@@ -290,10 +290,6 @@ public function testGetPlatformJsonGraph(): void
 
     public function testDeletePlatformTopologySuccess(): void
     {
-        $this->platformTopologyService
```

Comment:
what kind of warning is returned when this test is runned ?

Reply:
wouldn't removing this part make the test useless ?
as the test is meant to test the deletePlatform method

Reply:
this method returns void, not null
So we don't need to override the default mock

Reply:
```
There were 2 warnings:

1) Tests\Centreon\Application\Controller\PlatformTopologyControllerTest::testDeletePlatformTopologySuccess
Trying to configure method "deletePlatform" which cannot be configured because it does not exist, has not been specified, is final, or is static

2) Tests\Centreon\Domain\PlatformTopology\PlatformTopologyServiceTest::testDeletePlatformTopologySuccess
Method deletePlatform may not return value of type NULL, its return declaration is ": void"
```